ray-operator/config/samples/ray_v1alpha1_rayjob.yaml

```diff
@@ -102,7 +102,7 @@ spec:
   #         image: rayproject/ray:2.9.0
   #         # If Command is not specified, the correct command will be supplied at runtime using the RayJob spec `entrypoint` field.
   #         # Specifying Command is not recommended.
-  #         # command: ["ray job submit --address=http://rayjob-sample-raycluster-v6qcq-head-svc.default.svc.cluster.local:8265 -- echo hello world"]
+  #         # command: ["ray job submit --address=http://$FQ_RAY_IP:8265 -- echo hello world"]
```

Comment:
Would you mind trying whether it actually works or not? I forgot why I divided the command into a slice of multiple strings instead of a slice with just one string in [rayjob_test.go](https://github.com/ray-project/kuberay/blob/c45d959a2e149b640f292b1304dcbbb8289486f8/ray-operator/test/e2e/rayjob_test.go#L168).

Reply:
The env injection works, but the original entrypoint was invalid. Updated to this:
```
command: ["sh", "-c", "ray job submit --address=http://$FQ_RAY_IP -- echo hello world"]
```